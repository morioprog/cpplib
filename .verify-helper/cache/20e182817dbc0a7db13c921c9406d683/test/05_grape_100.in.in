100
2 25 37
0
2 38 86
2 15 64
0
2 14 51
0
2 26 82
2 52 88
0
2 28 49
2 35 60
2 42 78
0
0
0
0
0
2 21 94
2 72 99
2 18 47
2 1 45
0
0
2 13 79
2 23 63
0
0
0
0
0
0
2 58 93
0
2 19 46
0
2 8 68
0
0
0
2 36 65
0
0
0
2 11 75
2 48 91
0
0
0
2 41 92
2 12 30
2 43 44
0
0
2 20 74
0
2 80 96
0
2 67 70
2 27 76
2 59 95
2 57 81
2 5 66
2 32 55
2 10 97
0
0
2 31 50
0
2 40 77
0
0
0
2 39 98
0
0
2 24 89
0
2 29 83
2 9 73
0
2 4 56
2 2 6
2 7 71
0
0
2 62 85
0
2 17 54
0
1 53
2 33 61
2 34 84
0
0
0
2 3 87
0
2 16 69
2 22 90
100
1 17
0 55 75
0 13 7
1 24
0 10 62
1 1
0 79 66
1 4
1 28
0 32 28
0 68 26
0 35 19
0 52 9
1 78
0 24 85
0 48 56
0 9 3
0 48 82
1 86
0 60 15
1 44
1 13
0 67 45
1 38
0 52 99
0 30 74
0 66 38
0 21 63
1 75
0 94 9
0 33 98
0 99 22
1 93
1 3
0 30 4
0 81 56
0 91 86
1 28
0 63 15
1 26
1 52
0 7 61
0 92 59
1 91
0 88 38
1 43
1 96
1 5
1 53
1 21
1 80
0 93 97
0 70 23
1 22
0 87 35
1 79
1 45
1 24
0 11 43
0 61 39
0 66 69
0 24 42
0 5 44
0 1 11
0 75 39
0 68 78
0 15 9
1 12
0 16 50
1 58
1 56
1 62
1 69
0 99 63
1 56
0 43 10
1 34
1 58
0 4 37
0 71 45
0 83 28
0 85 23
0 67 79
0 81 5
1 80
0 45 56
0 54 99
1 22
0 4 34
1 61
0 27 33
0 14 69
0 69 54
0 82 73
0 96 7
0 30 87
0 27 42
1 82
1 37
1 77
