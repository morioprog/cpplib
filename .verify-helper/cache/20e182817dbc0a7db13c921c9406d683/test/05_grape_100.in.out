0
0
0
66
128
28
28
28
135
73
73
28
302
506
88
362
464
149
558
149
701
416
558
701
315
416
249
88
43
726
149
444
726
847
43
838
1199
932
329
0
678
